drivers/pinctrl/pinctrl_gd32_af.c

```diff
@@ -21,6 +21,7 @@ BUILD_ASSERT((GD32_OTYPE_PP == GPIO_OTYPE_PP) &&
 
 BUILD_ASSERT((GD32_OSPEED_2MHZ == GPIO_OSPEED_2MHZ) &&
 #if defined(CONFIG_SOC_SERIES_GD32F3X0) || \
+	defined(CONFIG_SOC_SERIES_GD32A50X) || \
```

Comment:
Update commit message of this commit. like gd32 pinctrl driver support gd32a50x series.